apollos-church-api/package.json

```diff
@@ -16,7 +16,7 @@
     "lint": "eslint ./src --ext .js"
   },
   "engines": {
-    "node": ">=10.19.x"
+    "node": ">=8.x.x"
```

Comment:
This can/should probably be backed out

Reply:
so you want me to undo that change? I'm confused